FingerBlox/app/src/main/java/org/fingerblox/fingerblox/CameraView.java

```diff
@@ -98,4 +101,13 @@ protected void fixFocusToggle() {
         }
         mCamera.setParameters(parameters);
     }
+
+    public Camera.Parameters getCameraParameters() {
+        try {
+            return mCamera.getParameters();
+        } catch (NullPointerException e) {
+            Log.i(TAG, "Could not retrieve camera parameters. Camera in CameraView == NULL");
```

Comment:
Not true, this happens when the camera is still initializing and when the phone locked. Putting the code in onResume(), onCameraViewStarted() or any of the other functions causes this Exception to be thrown immideately (as the camera is started later than the view).